### ISSUE TITLE
Perhaps a narrower focus?

### ISSUE THREAD

**Issue by zachlysobey** [guipdutra/awesome-geek-podcasts#89]
I have actually creating a similar list and was simultaneously excited and disappointed to find yours.

https://github.com/zachlysobey/software-development-podcasts/

awesome-geek-podcasts is, so far at least, significantly better in almost every way.

The only thing that bothers me a bit is that it seems to be 95% about software and imposes no real restriction on the kinds of podcasts that can be added (except that they are awesome and appeal to geeks?)

Hardcore History, for example, is an amazing podcast, but I feel as though it doesn't really fit (with _my_ vision at least). I can see it becoming a "slippery slope" where pretty much anything can and will be added here. In my opinion this detracts from the list.

Long story short, I'm curious if this projects goals align with mine, or if I should just maintain my project separately and essentially merge in all of your hard work. I am not against shutting mine down in favor of this one and contributing here, but I just want to be on the same page.

**Comment by ayr-ton** [comment]
@zachlysobey I don't think is an issue to have things like "Hardcore History", we have things like "radiolabs" and other related to pop culture.

**Comment by ayr-ton** [comment]
I will leave this one closed for now. And when we got new commits/comments/PRs we could reopen o/
